### PR TITLE
fix(withAuthenticator): RN Set default usernameAttributes to username and autofill the username in ConfirmSignUp Page.

### PR DESCRIPTION
**Pull request by chintannp**
#### Description of changes
Set default `usernameAttributes` to username when no `usernameAttributes` is passed by the user. This will automatically store the usernameAttribute in the state and will autofill the username field. Additionally, it will automatically enable the **Resend code** button as the state will not be empty or undefined for `username` property.

#### Issue #, if available

#9447 

#### Description of how you validated changes
Built an Expo app by using `withAuthenticator` component and testing SignIn, SignUp & ConfirmSignUp functionality.

#### Checklist

- [x] PR description included
- [x] `yarn test` passes
- [ ] Tests are [changed or added](https://github.com/aws-amplify/amplify-js/blob/main/CONTRIBUTING.md#steps-towards-contributions)
- [ ] Relevant documentation is changed or added (and PR referenced)

By submitting this pull request, I confirm that my contribution is made under the terms of the Apache 2.0 license.

### CODE REVIEW

File: packages/aws-amplify-react-native/src/Auth/Authenticator.tsx [hunk 1]
```diff
@@ -219,11 +204,9 @@ export default class Authenticator extends React.Component<
 		// otherwise if truthy, use the supplied render prop
 		// otherwise if falsey, use EmptyContainer
 		const ContainerWrapper: any =
-			this.props.container === undefined
-				? Container
-				: this.props.container || EmptyContainer;
+			this.props.container === undefined ? Container : this.props.container || EmptyContainer;
 
-		const { hideDefault, signUpConfig, usernameAttributes } = this.props;
+		const { hideDefault, signUpConfig, usernameAttributes = 'username' } = this.props;
```
Comment (chintannp) [inline packages/aws-amplify-react-native/src/Auth/Authenticator.tsx:209]: This is the only change that I have introduced with the PR. All other changes are related to formatting. The previous formatting was not as per the `.prettierrc.js` mentioned in the project. Hence, I think we should also fix the code formatting as well.

Comment (cshfang) [reply packages/aws-amplify-react-native/src/Auth/Authenticator.tsx:209]: Thanks for calling this out 👍

**Review by cshfang — APPROVED** [review]
Lgtm 🚀

**Review by calebpollman — APPROVED** [review]
LGTM 🚢

**Pull request merged**